src/views/Main/Main.tsx

```diff
@@ -92,6 +92,13 @@ export function MainView() {
         initReportsIcon(history),
       );
     }
+
+    if (userModel?.hasFeature(AppFeature.BIVARIATE_COLOR_MANAGER)) {
+      import('~features/bivariate_color_manager/').then(
```

Comment:
Just thinking about the reason of using lazy import here. Now the logic is: we load **index** bundle, it checks features and if color management feature is available then it loads another bundle which is only consists of one file **src/features/bivariate_color_manager/index.tsx** after this file run its logis app redirects user to new route **./bivariate-manager** after that our router will load another bundle with actual bivariate color config (we also use lazy loading in router, so another bundle will be created consequently). The question here is: do we really need to split **bivariate_color_manager/index.tsx** into a separate bundle?

Reply:
good point

Reply:
You're right! This logic looks redundant. 
I also see we have the same situation with  import('~features/reports/').
So I'll remove lazy-loading for it also.